nexus/db-model/src/instance.rs

```diff
@@ -544,4 +544,8 @@ pub struct InstanceUpdate {
     /// set the instance's auto-restart policy to `NULL`.
     #[diesel(column_name = auto_restart_policy)]
     pub auto_restart_policy: Option<InstanceAutoRestartPolicy>,
+
+    pub ncpus: InstanceCpuCount,
+
+    pub memory: ByteCount,
```

Comment:
These should probably have doc comments so that they're documented in the OpenAPI spec?

Reply:
This is the model, not the request body, so it doesn't matter for OpenAPI.

Reply:
Ah, whoops! @david-crespo is right, never mind --- looks like the request body type already has them.